ci-operator/step-registry/aws/deprovision/route53/private-hosted-zone/aws-deprovision-route53-private-hosted-zone-commands.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+REGION="${LEASED_RESOURCE}"
+HostedZoneId="${SHARED_DIR}/hosted_zone_id"
+
+echo "Deleting AWS route53 hosted zone"
```

Comment:
nit: put this right before the `delete-hosted-zone` command, so folks don't get confused about whether or not it happened in the `File ${HostedZoneId} does not exist.` case?

Reply:
Done